editors/neovim/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup               github 1.0
 PortGroup               cmake 1.1
 
 github.setup            neovim neovim 0.10.0 v
-revision                0
+revision                1
```

Comment:
This only bumps the revision of the main neovim port. Since the pre-configure block you added is in the global part of the port is also affects the neovim-devel subport which needs toe revbumped separately below.